src/styles/blocs/card/stars_card.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
```

Comment:
you can write this shorter, on the same line

```
    background-image: url(/src/images/star.svg);
    background-position: center;
    background-repeat: no-repeat
```

Reply:
Ready)